components/atom/input/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@s-ui/react-atom-input",
+  "version": "1.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "npx rimraf ./lib && npx mkdirp ./lib && npm run build:js && npm run build:styles",
+    "build:js": "../../../node_modules/.bin/babel --presets sui ./src --out-dir ./lib",
+    "build:styles": "../../../node_modules/.bin/cpx './src/**/*.scss' ./lib"
+  },
+  "dependencies": {
+    "@s-ui/component-dependencies": "1",
+    "@s-ui/react-form-input": "1.6.0",
+    "@s-ui/react-molecule-field": "1.1.0"
```

Comment:
Please set only the major version in order to not have to update this package everytime